### PR TITLE
GH37586: Adds netobserv notes to web console section

### PR DESCRIPTION
**Pull request by opayne1**
Relates to GH issue #37586 [comment ](https://github.com/openshift/openshift-docs/issues/37586#issuecomment-1033888827) 

- Applies to `enterprise-4.10` only
- preview link

### CODE REVIEW

**Comment by opayne1** [comment]
@jotak took this on for Neal. PTAL. Thank you!

**Review by jotak — APPROVED** [review]
Looks good to me, thanks!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -184,6 +184,18 @@ With this update, you can customize workload notifications on the *User Preferen
...
+==== Listing pods targeted by network policies
+When using the network policy functionality in the {product-title} web console, the pods that are affected by a policy are listed. The list updates as the combined namespace and pod selectors in the following policy sections are modified:
+* Peer definition
+* Rule definition
+* Ingress
+* Egress
+
+[id="switching-between-yaml-and-web-console-form"]
+==== Switching between YAML and a web console form
+With this update, you can now switch between YAML and a web console form in the web console while saving any changes made.
```
Comment (mjpytlak) [inline release_notes/ocp-4-10-release-notes.adoc:197]:
```suggestion
With this update, you can now switch between YAML and a web console form in the {product-title} web console while saving any changes made. 
```
I stumbled over the last part of the sentence. Are you saying that changes are saved automatically when you switch between YAML a web console form?

Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc]: yes, updated. Thanks!

**Comment by opayne1** [comment]
miscommunication: already documented/merged. See https://github.com/openshift/openshift-docs/pull/42108

**Pull request closed without merging**